### PR TITLE
When using it on Image component, getting exception for Android

### PR DESCRIPTION
**Pull request by jhonatanfernando**
For Android version, getting exception when is used on Image component.

### CODE REVIEW

File: TouchEffectSample.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.136
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29709.97
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TouchEffectSample.Android", "TouchEffectSample.Android\TouchEffectSample.Android.csproj", "{9EED875F-F3C4-4753-9064-39DF7CD222AA}"
 EndProject
```
Comment (AndreiMisiukevich) [inline TouchEffectSample.sln:6]: Please discard this file

File: TouchEffectSample.iOS/TouchEffectSample.iOS.csproj [hunk 2]
```diff
@@ -160,6 +160,8 @@
     <ProjectReference Include="..\TouchEffect.iOS\TouchEffect.iOS.csproj">
       <Project>{78144B76-1773-47C2-B4D1-5351BEC6867D}</Project>
       <Name>TouchEffect.iOS</Name>
+      <IsAppExtension>false</IsAppExtension>
```
Comment (AndreiMisiukevich) [inline TouchEffectSample.iOS/TouchEffectSample.iOS.csproj]: discard this file too

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 3]
```diff
@@ -16,15 +16,15 @@
 using static System.Math;
 
 [assembly: ResolutionGroupName(nameof(TouchEffect))]
-[assembly: ExportEffect(typeof(PlatformTouchEff), nameof(TouchEff))]
+[assembly: ExportEffect(typeof(PlatformTouchEff), nameof(TouchEff))]
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs]: discard this line too

**Review by jhonatanfernando — COMMENTED** [review]
Fixed.

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 4]
```diff
@@ -16,15 +16,14 @@
 using static System.Math;
 
 [assembly: ResolutionGroupName(nameof(TouchEffect))]
-[assembly: ExportEffect(typeof(PlatformTouchEff), nameof(TouchEff))]
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs:19]: Hey) I asked to discard changes in that line, not remove the whole line

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 5]
```diff
@@ -16,15 +16,14 @@
 using static System.Math;
 
 [assembly: ResolutionGroupName(nameof(TouchEffect))]
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs:18]:
```suggestion
[assembly: ResolutionGroupName(nameof(TouchEffect))]
[assembly: ExportEffect(typeof(PlatformTouchEff), nameof(TouchEff))]
```

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 6]
```diff
@@ -97,13 +101,18 @@ protected override void OnDetached()
             catch (ObjectDisposedException)
             {
                 //suppress exception
+            }
+            catch(Exception)
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs:105]: remove this empty catch

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 7]
```diff
@@ -97,13 +101,18 @@ protected override void OnDetached()
...
+            {
+
             }
             _isHoverSupported = false;
         }
 
         private void OnTouch(object sender, AView.TouchEventArgs e)
         {
             e.Handled = true;
+
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs:115]: remove empty line

**Pull request merged**